README.md

```diff
@@ -62,6 +62,7 @@ Make a new Hyperbee instance. `core` should be a [Hypercore](https://github.com/
 {
   keyEncoding: 'binary', // "binary" (default), "utf-8", "ascii", "json", or an abstract-encoding
   valueEncoding: 'binary' // Same options as keyEncoding like "json", etc
+  onlyIfChanged: true // put a value only if it means a change in the db
```

Comment:
Alternative name: `changeOnly`

Reply:
Or `casChanges`

Reply:
I like the boolean option, but just maybe a different API could be a `cas` option in the constructor, so user can pass its global comparator